src/api.js

```diff
@@ -0,0 +1,16 @@
+const TODOS_URL = 'https://mate-api.herokuapp.com/todos/';
+const USERS_URL = 'https://mate-api.herokuapp.com/users/';
```

Comment:
```suggestion
const BASE_URL = 'https://mate-api.herokuapp.com/';
```
And then add either /todos or /users in your 'getTodos'/'getUser' methods